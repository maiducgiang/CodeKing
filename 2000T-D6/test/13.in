67
1954 63204 11287 12514 2243 84724 69722 69674 74023 70896 97416 86193 5271 26934 22466 22437 25180 78952 28641 98180 62449 64847 58814 85034 95471 2142 22568 34907 6093 25117 44706 21568 11903 70005 41361 51911 95015 59508 38862 71645 43497 48849 89708 18196 43545 29866 9682 45609 44131 66299 78865 75696 48614 60319 8145 30280 91899 95502 33472 63569 76325 50052 47741 4155 14594 12597 47415 