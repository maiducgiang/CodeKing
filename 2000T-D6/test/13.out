977 15801 -1 6257 -1 21181 34861 34837 937 4431 12177 9577 1757 13467 11233 277 6295 9869 9547 24545 317 3413 29407 42517 -1 1071 2821 521 677 -1 22353 337 -1 23335 13787 2257 19003 14877 19431 14329 179 16283 22427 4549 14515 14933 4841 15203 -1 397 15773 4731 24307 1231 905 3785 10211 47751 523 5779 3053 12513 -1 1385 7297 4199 15805 
2 2 -1 2 -1 2 2 2 79 2 2 3 3 2 2 3 2 2 3 2 197 19 2 2 -1 2 2 67 3 -1 2 2 -1 3 3 23 5 2 2 5 3 3 2 2 3 2 2 3 -1 167 5 2 2 7 3 2 3 2 2 11 5 2 -1 3 2 3 3 